src/lib/routes/admin-api/user/pat.ts

```diff
@@ -48,7 +52,17 @@ export default class PatController extends Controller {
                 openApiService.validPath({
                     tags: ['Personal access tokens'],
                     operationId: 'getPats',
-                    responses: { 200: createResponseSchema('patsSchema') },
+                    summary: 'Get all current users Personal Access Tokens.',
```

Comment:
May sound a bit unclear 🤔 Would this be clearer?
```suggestion
                    summary: 'Get all Personal Access Tokens for the current user.',
```